examples/dataflow-python-examples/README.md

```diff
@@ -284,3 +286,23 @@ Ready to dive deeper?  Check out the complete code.
 The example using side inputs is [here](dataflow_python_examples/data_lake_to_mart.py) and the example using CoGroupByKey is
 [here](dataflow_python_examples/data_lake_to_mart_cogroupbykey.py).
 
+## Slowly Changing Side Input
+This example demonstrates the concept of referencing multiple forms of side input data in GCS with main input data loaded from pubsub, and triggering the refresh of side input data via PubSub notification on demand.
+
+This pipeline contains 4 steps:
+1. Read Primary events from main input pubsub subscription over a fixed window.
+2. Read GCS file base path containing latest side input data from side input pubsub subscription.
+3. (Re) Load Side input data from corresponding side input sub path under the base path received in step 2
```

Comment:
It would be good to describe a motivating use case and call out that one might hook this up to a object change notification on the look-up file.